src/app/bitstream-page/clarin-license-agreement-page/clarin-license-agreement-page.component.ts

```diff
@@ -204,6 +208,10 @@ export class ClarinLicenseAgreementPageComponent implements OnInit {
     this.router.navigate([getItemPageRoute(this.item$?.value)]);
   }
 
+  private isDownloadingZIP() {
+    return this.router.routerState.snapshot.url.endsWith('/zip');
```

Comment:
Is this really a good way to detect zip files? what if someone omits extension and file is e.g. txt file called `howtounzip` or something similar?